Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11707,6 +11717,17 @@ a_Player:OpenWindow(Window);
 				{
 					Notes = "Reloads the player's rank, message visuals and permissions from the {{cRankManager}}, based on the player's current rank.",
 				},
+				LoginSetGameMode =
+				{
+					Returns =
+					{
+						{
+							Name = "eGameMode",
+							Type = "eGameMode",
+						},
+					},
+					Notes = "Sets the current gamemode, doesn't check validity, doesn't send update packets to client.",
```

Comment:
Why is it called `Login`...? And does it even have to be exported to Lua? Doesn't feel like it, sounds more like some internals bleeding out to API.